Project-I-Social-Card/social-card/src/components/CardComponents/CardBanner.js

```diff
@@ -1,2 +1,11 @@
 import React from 'react';
 import './Card.css';
+
+function CardBanner() {
+    return (
```

Comment:
You need to wrap multiple elements in JSX in a container, usually a div that wraps everything being returned